### PR TITLE
V2.14-RUN-10061-Ephemeral-volumes

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

File: docs/Researcher/user-interface/trainings.md [hunk 1]
```diff
@@ -31,8 +32,19 @@ To add a training:
...
+7. Open the *Volume* pane, and press *Volume* to add a volume to your training.
+   1. Select the *Storage class* from the dropdown.
+   2. Select the *Access mode* from the dropdown.
+   3. Enter a claim size, and select the units.
+   4. Select a *Volume system*, mode from the dropdown.
+   5. Enter the *Container path* for volume target location.
+   6. Select a *Volume persistency.
+8. In the *Data sources* pane, press *add a new data source*. For more information, see [Creating a new data source](workspaces/create/create-ds.md) When complete press, *Create Data Source*.
+9. In the *General* pane, add special settings for your training (optional):
+   1. Press *Auto-deletion* to delete the training automatically when it either completes or fails. You can configure the time in days, hours, minuets, and seconds.
```
Comment (gal-revach) [inline docs/Researcher/user-interface/trainings.md:44]:
```suggestion
   1. Press *Auto-deletion* to delete the training automatically when it either completes or fails. You can configure the timeframe in days, hours, minuets, and seconds.
```

File: docs/home/whats-new-2-14.md [hunk 2]
```diff
@@ -29,6 +29,28 @@ When upgrading the system, previous access and authorizations that were configur
...
+<!-- RUN-10335/RUN-10510 Node port command line -->
+
+<!-- RUN-10575/RUN-10579 Add numeric rules in the policy to GPU memory, CPU memory & CPU -->
+* We are pleased to announce that we have increased the flexibility when creating policies. The improved flexibility includes the abililty to allocate a `min` and a `max` value for CPU and GPU memory.
+
+<!-- RUN-11421/RUN-11508 Consumption report cost and bugs -->
+* We are pleased to announce that we have added an additional table to the Consumption dashboard. This table contains the consumption per department and the cost per department. For more information, see [Consumption dashboard]()
+
+<!-- RUN-8586/RUN-11777 Auto delete completed/failed jobs -->
+
```
[the remark this answers is not in the document]
Comment (jasonnovichRunAI) [reply docs/home/whats-new-2-14.md:52]: What's new is not ready yet. Will be filled in later.

**Review by shayasoolin — APPROVED** [review]

File: docs/Researcher/cli-reference/runai-submit.md [hunk 3]
```diff
@@ -105,6 +105,10 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
 >
 > Interactive preemptible jobs can be scheduled above guaranteed quota but may be reclaimed at any time.
 
+#### --auto-deletion-time-after-completion
+>
+> The length of time after which a completed or failed job is automatically deleted. Configured in seconds, minutes, or hours (for example 5s, 2m, or 3h). If set to 0, the job will be deleted immediately after completing or failing.
```
Comment (gal-revach) [inline docs/Researcher/cli-reference/runai-submit.md:110]:
```suggestion
> The timeframe after which a completed or failed job is automatically deleted. Configured in seconds, minutes, or hours (for example 5s, 2m, or 3h). If set to 0, the job will be deleted immediately after completing or failing.
```

File: docs/Researcher/user-interface/trainings.md [hunk 4]
```diff
@@ -31,8 +32,19 @@ To add a training:
...
+7. Open the *Volume* pane, and press *Volume* to add a volume to your training.
+   1. Select the *Storage class* from the dropdown.
+   2. Select the *Access mode* from the dropdown.
+   3. Enter a claim size, and select the units.
+   4. Select a *Volume system*, mode from the dropdown.
+   5. Enter the *Container path* for volume target location.
+   6. Select a *Volume persistency.
+8. In the *Data sources* pane, press *add a new data source*. For more information, see [Creating a new data source](workspaces/create/create-ds.md) When complete press, *Create Data Source*.
+9. In the *General* pane, add special settings for your training (optional):
+   1. Press *Auto-deletion* to delete the training automatically when it either completes or fails. You can configure the timeframe in days, hours, minuets, and seconds.
```
Comment (gal-revach) [inline docs/Researcher/user-interface/trainings.md:44]:
```suggestion
   1. Press *Auto-deletion* to delete the training automatically when it either completes or fails. You can configure the timeframe in days, hours, minuets, and seconds. If the timeframe is set to 0, the training will be deleted immediately after it completes or fails.
```

File: docs/Researcher/user-interface/workspaces/create/workspace-v2.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+7. Open the *Volume* pane, and press *Volume* to add a volume to your workspace.
+   1. Select the *Storage class* from the dropdown.
+   2. Select the *Access mode* from the dropdown.
+   3. Enter a claim size, and select the units.
+   4. Select a *Volume system*, mode from the dropdown.
+   5. Enter the *Container path* for volume target location.
+   6. Select a *Volume persistency.
+8. Select a data source from the tiles. If your data source is not listed, use the *Search compute resources* box to find it. Press *New data source* to create a new data source if needed.
+9. In the *General* pane, add special settings for your workspace:
+   1. Press *Auto-deletion* to delete the workspace automatically when it either completes or fails. You can configure the timeframe in days, hours, minuets, and seconds.
```
Comment (gal-revach) [inline docs/Researcher/user-interface/workspaces/create/workspace-v2.md:31]:
```suggestion
   1. Press *Auto-deletion* to delete the workspace automatically when it either completes or fails. You can configure the timeframe in days, hours, minuets, and seconds. If the timeframe is set to 0, the workspace will be deleted immediately after it completes or fails.
```

**Pull request merged**